src/lib/singularity.h

```diff
@@ -158,14 +158,4 @@
     extern int singularity_fork_exec(char **argv);
 
 
-
-
-    extern int singularity_config_open(char *config_path);
-    extern void singularity_config_close(void);
-    extern void singularity_config_rewind(void);
-
-    extern char *singularity_config_get_value(char *key);
-    extern int singularity_config_get_bool(char *key, int def);
-
```

Comment:
I'm not sure what the benefit is, but whatever we do decide on we should make sure it's consistent across all files and functions